src/shared/bg_public.h

```diff
@@ -133,6 +133,12 @@ struct playerState_t
 	int           misc[ MAX_MISC ]; // misc data
 };
 
+// the possibily and the cost of evolving to an alien form
+struct evolveInfo_t {
+	bool canEvolve;
```

Comment:
The name seems misleading. `formUnlocked` would make more sense

Reply:
updated the name